test/multi-runtime-services.test.ts

```diff
@@ -0,0 +1,103 @@
+import * as assert from 'assert';
+import * as path from 'path';
+import { cleanOpenRuntimes, launchAndConnect } from './multi-runtime-utils';
+import { delayPromise } from './delay-promise';
+import * as sinon from 'sinon';
+import * as fs from 'fs';
+
+describe ('Multi Runtime Services', () => {
+    beforeEach(async () => {
+        await cleanOpenRuntimes();
+    });
+
+    after(async () => {
+        await cleanOpenRuntimes();
+    });
+
+    // tslint:disable-next-line
+    describe('Multi Runtime with External Provider', function () {
+
+        it('Should work in multi runtime with an external Provider', function(done: any) {
+            // tslint:disable-next-line no-invalid-this
```

Comment:
These `tslint:disable-next` can go if you add the following line: 
`/* tslint:disable:no-invalid-this no-function-expression insecure-random mocha-no-side-effect-code no-empty */`

similar to the multi-runtime tests: https://github.com/HadoukenIO/js-adapter/blob/develop/test/multi-runtime-events.test.ts